site/app/templates/grading/electronic/GradingPanelHeader.twig

```diff
@@ -61,7 +61,7 @@
             {{ self.renderPanelPositionSelector('student_info') }}
             </span>
         {% endif %}
-        <span class="grade-panel" id="solution_ta_notes_btn">
+        <span class="grade-panel" id="solution_ta_notes_btn" data-testid="solution_ta_notes_btn">
```

Comment:
remove if you are not using this

Reply:
I added these to prevent this problem happening again in the future. Adding a test-id to an element is free,  and can possibly useful for future tests, like the ones I'm writing for https://github.com/Submitty/Submitty/pull/10635. Do you still think I should remove it?

Reply:
whenever i add some additional testid in my pr cameron requested me to remove it , and also one more point we are more consistent with hypens in data-testid not underscore

Reply:
This PR is adding test-ids for #10635, so it is an exception to the 'no unused' 'rule'